raft-tech/TANF-app
# #1858 Kibana access + changes for fully local development

- Enables direct frontend/backend communication sans Login.gov/Cloud.gov
 - Drives off new REACT_APP_DEVAUTH env var
 - Pre-configures and disables frontend auth functionality
 - Testing based on new dev user
   - Install via web: ./manage.py generate_dev_user


## How to Test
Start frontend and backend locally with REACT_APP_DEVAUTH set.
Create the dev user in a web container shell: ./manage.py generate_dev_user
Open http://localhost:3000/data-files in browser.
Submit a data file normally.
Verify with logs and postgres DB query.
